### ISSUE TITLE
Touch support

### ISSUE THREAD

**Issue by fracz** [drewjbartlett/vue-switches#11]
When you open a switch on a mobile device, you should be able to change its state with swipe gesture.

**Comment by fracz** [comment]
When you open a switch on a mobile device, you should be able to change its state with swipe gesture.

**Comment by drewjbartlett** [comment]
Agreed. I haven't had the chance to add this yet. Feel free to make a PR
